x-pack/filebeat/module/aws/vpc/test/accept-reject-traffic.log-expected.json

```diff
@@ -0,0 +1,76 @@
+[
+    {
+        "@timestamp": "2014-12-14T04:07:50.000Z",
+        "aws.vpc.account_id": "123456789010",
+        "aws.vpc.action": "ACCEPT",
+        "aws.vpc.bytes": "4249",
+        "aws.vpc.dstaddr": "172.31.16.21",
+        "aws.vpc.dstport": "22",
+        "aws.vpc.end": "1418530070",
+        "aws.vpc.interface_id": "eni-1235b8ca123456789",
```

Comment:
One of the challenging things about using these logs is associating the ENI back to an actual instance-id. Is there anything that would allow us to call [DescribeNetworkInterfaces](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeNetworkInterfaces.html) to store that info to enrich these events?

Reply:
@andrewkroh Hmm I'm not sure how to add actual code into filebeat to reach out to AWS API to get these information. But definitely something very useful though!! The only thing I can think of right now is to use javascript somehow to make these api calls and use the response to enrich the event.

I would prefer making this a separate PR. Is that OK for you?

Reply:
It's not necessarily something we have to tackle. It was more of an open question and to see if any enrichments like this have come up in past discussions or if you were thinking about it.

Reply:
I think we have a similar need for some enrichments in all cloud providers, where we would want to retrieve metadata (like extra info/labels/tags from monitored resources). So far we haven't gone into this but it would be interesting to have a way to solve that.

Reply:
One thought that came to mind for this particular case was to index the ENI metadata periodically. Then use the Elasticsearch [Enrich Ingest Node processor](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/ingest-enriching-data.html#create-enrich-source-index) to add the metadata to the associated documents at ingest time.

Reply:
https://github.com/elastic/beats/issues/14627 I created a separate issue for tracking this work.

Reply:
+1 on moving the conversation there. I had a look to the enrich processor, it's a pity that you cannot update the enrich index on the fly. This makes it really difficult to use it for highly dynamic resources (like for instance containers or even EC2 instances)